src/server/game/Quests/QuestDef.cpp

```diff
@@ -285,6 +285,9 @@ bool Quest::IsAllowedInRaid(Difficulty difficulty) const
     if (IsRaidQuest(difficulty))
         return true;
 
+    if (SpecialFlags & QUEST_SPECIAL_FLAGS_ALLOWED_IN_RAID)
```

Comment:
`if ((SpecialFlags & QUEST_SPECIAL_FLAGS_ALLOWED_IN_RAID) != 0)`

There was a compiler that had a warning due to implicit conversion to bool, better make it explicit